xml/release-notes.xml

```diff
@@ -300,6 +300,67 @@
        </step>
        </procedure>
    </sect3>
+   <sect3 xml:id="sec.upgrade.422.shorewall">
+    <title>Shorewall upgrade</title>
+    <para>
+      In &opensuseleap; 42.2, Shorewall was upgrade to last stable 5.1x series.
+      <package>shorewall</package> and <package>shorewall6</package> on upgrade
```

Comment:
Yeap it's still in adi:97 also the last revision of the package for Tumbleweed is in staging area.
As I'm going on holidays tomorrow, and there's always so much things to do afterwards that I'll prefer to submit earlier.
So from others comments, I've some minor fixes to add to the PR, and also feel free to merge it only when the package is in 42.3.
See others comments.